src/test/java/org/spdx/library/ListedLicensesTest.java

```diff
@@ -116,7 +116,7 @@ public void testGetListedExceptionByIdReturnsNull() throws InvalidSPDXAnalysisEx
 		assertNull(result);
 	}
 
-	public void testGetExceptionbyIdLocal() throws InvalidSPDXAnalysisException {
```

Comment:
Same here - should deprecate (or not fix)

Reply:
This one is public, but in a test class, so I am not sure that it's necessary.

Reply:
I think it's safe to just rename the test cases.

Reply:
By the way, Junit 5 no longer requires test methods to be public, so the fact that it is public in the first place is just due to a requirement of Junit 4.

Reply:
Correct - no need to worry about the test case code